### DISCUSSION TITLE
[Feature]: Edge Scripting APIs

### DISCUSSION THREAD

**Discussion by jlarmstrongiv** [ToshY/BunnyNet-PHP#140] [General]
### Description

<img width="965" alt="image" src="https://github.com/user-attachments/assets/e8c4c8aa-de6c-45d9-826a-e1ba080afdf9">

**Comment by ToshY** [comment]
Hey @jlarmstrongiv 👋 

That seems to be in closed beta testing. As there doesn't seem to be (public) API specifications for this in the [documentation](https://docs.bunny.net/reference/bunnynet-api-overview) yet, there's not much I can do right now (unless you know where to find the specs by any chance).

When it becomes generally available and the API specs are known, I could start on it myself. People are more than welcome to contribute and whip up a draft PR for this if they know more about this new feature.

**Reply by ToshY** [reply]
Until further notice I've changed this to a discussion.

**Reply by ToshY** [reply]
@jlarmstrongiv This will be added in #144

**Comment by ToshY** [comment]
Closed with #145 ; will be released in `5.0`
